Runtimes/Supplemental/Synchronization/cmake/caches/Vendors/Apple/x86_64-MacOSX.cmake

```
@@ -0,0 +1,18 @@
+if(NOT DEFINED CMAKE_OSX_DEPLOYMENT_TARGET)
+  message(SEND_ERROR "CMAKE_OSX_DEPLOYMENT_TARGET not defined")
+endif()
+
+if(NOT DEFINED SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET)
+  message(WARNING "SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET not defined")
```

Maybe make this an error?

```suggestion
  message(SEND_ERROR "SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET not defined")
```